### DISCUSSION TITLE
Trouble Setting up the Netbox DB Schema and Creating a Superuser.

### DISCUSSION THREAD

**Discussion by jacen2110** [netbox-community/netbox#11811] [Help Wanted!]
I have been these instructions (https://writemem.co.uk/install-netbox-a-guide-that-works/#:~:text=Netbox%20Install%20%E2%80%93%20Step-by-step%20Guide%201%20STEP%201%3A,STEP%208%3A%20Start%20your%20NGINes%20...%20More%20items) on how to install Netbox and when I get to step 7, I am thrown the following error:

kali㉿kali)-[/opt/netbox/netbox]
└─$ python3 manage.py migrate                      
Operations to perform:
  Apply all migrations: admin, auth, circuits, contenttypes, dcim, django_rq, extras, ipam, sessions, social_django, taggit, tenancy, users, virtualization, wireless
Running migrations:
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/dist-packages/django/db/backends/utils.py", line 87, in _execute
    return self.cursor.execute(sql)
           ^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.InsufficientPrivilege: permission denied for schema public
LINE 1: CREATE TABLE "django_migrations" ("id" bigint NOT NULL PRIMA...

Seems like I can't connect to the DB for some reason.

**Comment by markkuleinio** [comment]
https://github.com/netbox-community/netbox/discussions/11148#discussioncomment-4787188

**Reply by candlerb** [reply]
A better answer is at #11314

```
ALTER DATABASE netbox OWNER TO netbox;
```

See also https://www.cybertec-postgresql.com/en/error-permission-denied-schema-public/

If you are using Postgres >=15 then you might need

```
GRANT ALL ON SCHEMA public TO netbox;
```
